staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -1732,6 +1732,11 @@ type VolumeMount struct {
 	// This field is beta in 1.10.
 	// +optional
 	MountPropagation *MountPropagationMode `json:"mountPropagation,omitempty" protobuf:"bytes,5,opt,name=mountPropagation,casttype=MountPropagationMode"`
+	// Expanded path within the volume from which the container's volume should be mounted.
```

Comment:
How is the expansion done?
What component will do the expansion?
What sources of environment variables will be used for the expansion?

Reply:
I think it'd be better to describe this as "like SubPath, but with variable expansion immediately before use" or something?

Reply:
Amended description